packages/x-charts/src/ChartsTooltip/utils.tsx

```diff
@@ -1,3 +1,4 @@
+'use client';
```

Comment:
I thaught the CI was catching this kind of error

Reply:
It is, these were caught by eslint after deps update. 🤔